test/buy-tshirt.spec.ts

```diff
@@ -17,25 +17,38 @@ describe('Buy a t-shirt', () => {
   const orderSummaryPage: OrderSummaryPage = new OrderSummaryPage();
   const paymentStepPage: PaymentStepPage = new PaymentStepPage();
   const productAddedModalPage: ProductAddedModalPage = new ProductAddedModalPage();
-  const productListPage: ProductListPage = new ProductListPage();
   const shippinStepPage: ShippinStepPage = new ShippinStepPage();
   const signInStepPage: SignInStepPage = new SignInStepPage();
   const summaryStepPage: SummaryStepPage = new SummaryStepPage();
+  const productAddedPage: ProductAddedPage = new ProductAddedPage();
 
-  it('then should be bought a t-shirt', async () => {
-    await browser.get('http://automationpractice.com/');
-    await menuContentPage.goToTShirtMenu();
-    await addressStepPage.addToCar();
-    await bankPaymentPage.proceedToCheckout();
-    await orderSummaryPage.proceedToCheckout();
-    await paymentStepPage.fillEmailPassword();
-    await paymentStepPage.registerEmail();
-    await productAddedModalPage.proceedToCheckout();
-    await shippinStepPage.selectCheckBoxAgreeTerms();
-    await productListPage.proceedToCheckout();
-    await signInStepPage.payByBankWire();
-    await summaryStepPage.confirmOrder();
-    await expect(summaryStepPage.mensajeText())
-      .toBe('Your order on My Store is complete.');
+  describe('then should be bought a t-shirt', () => {
+
+    beforeEach(async () => {
+      await browser.get('http://automationpractice.com/');
+    });
+
+    describe('this is a test', () => {
```

Comment:
este describe no dice mucho, this is a test no me indica nada, si vas a cambiarlo a varios describe entonces que sea uno por step, sino, entonces conserva un sólo describe

Reply:
lo cambie por el siguiente mensaje "this execute all the lines into the beforeEach"